WhenPushComesToShove/Assets/3-Scripts/LevelSystems/LevelManager.cs

```diff
@@ -68,6 +68,7 @@ void ShowRoom()
             return;
         }
 
+        //Is this valid anymore?
         damageEnabler.EnableDamage(currentRoomIndex > 0);
```

Comment:
Do you know where this is connected to? If worse comes to worse, we can comment it out and see what happens.